File: output/CodeGen/codenet/Java/Python/s799002881.py
import sys
import io

def main(args):
    input = io.TextIOWrapper(sys.stdin.buffer, encoding='utf-8')
    output = io.TextIOWrapper(sys.stdout.buffer, encoding='utf-8')
    n, d = input.readline().strip().split(' ')
    n = int(n)
    d = int(d)
    cnt = 0
    for i in range(n):
        x, y = input.readline().strip().split(' ')
        x = int(x)
        y = int(y)
        if (math.sqrt(x * x + y * y) <= d):
            cnt += 1
    output.write(str(cnt))

if __name__ == '__main__':
    main(sys.argv)

File: output/ChatGPT/codenet/Java/C++/s799002881.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int n;
    double d;
    cin >> n >> d;
    int cnt = 0;

    for(int i = 0; i < n; i++) {
        double x, y;
        cin >> x >> y;
        if(sqrt(x*x + y*y) <= d) {
            cnt++;
        }
    }
    cout << cnt << endl;


    return 0;
}